lms/static/certificates/sass/_components.scss

```diff
@@ -537,7 +537,7 @@
 // #FOOTER
 // ------------------------------
 .footer-app {
-    border-top: rem(4) solid palette(grayscale, xx-trans);
+    border-top: rem(4) solid transparentize(palette(grayscale, base), 0.95);
```

Comment:
+1 for andy's comment about `back` (as opposed to `light`) -- I signed off on the code review that introduced `back` but seeing it used in context here I agree that `back` makes more sense. Something to think about as we expand this feature (as you discussed [here](https://github.com/edx/edx-platform/pull/13144#issuecomment-237218487), @clrux).